app/Castle/Knightclass.py

```diff
@@ -0,0 +1,52 @@
+from __future__ import annotations
+
+
+class Knight:
+    def __init__(self, name: str, power: int, hp, armour, weapon, potion=None) -> None:
```

Comment:
Consider adding type annotations for `hp`, `armour`, and `weapon` parameters in the constructor to improve code clarity and maintainability. This aligns with the checklist's recommendation to use annotations as a good practice.